com.fyralabs.Exponential.metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.fyralabs.Exponential</id>
+  <name>Exponential</name>
+  <developer id="com.fyralabs">
+    <name>Fyra Labs</name>
+  </developer>
+  <summary>A purpose-built tool for planning and building products</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
since this is proprietary and there is no further information we assume it's not redistributable so 'extra-data' must be used.

Reply:
The Linear team is comfortable with us redistributing, as long as the app metadata doesn't use the Linear branding. I can use extra-data though.

Reply:
Unless they have a license written down nobody can verify that.

Reply:
understandable, let me update it